PROTO_tests/tests/random_test.py

```diff
@@ -47,70 +52,75 @@ def test_integers(self):
         assert int_arr.to_list() == same_seed_int_arr.to_list()
 
         # verify within bounds (lower inclusive and upper exclusive)
+
         rng = ak.random.default_rng()
         bounded_arr = rng.integers(-5, 5, 1000)
         assert all(bounded_arr.to_ndarray() >= -5)
         assert all(bounded_arr.to_ndarray() < 5)
 
-    def test_shuffle(self):
+    @pytest.mark.parametrize("data_type",INT_FLOAT)
```

Comment:
Do we want to add `size` as a parameter to these?

Reply:
I considered that, but I don't really think it adds anything.  In fact, there are several of our tests where it seems that way to me.

Reply:
But Tess and I were just talking, and the conversation reminded me that some of our obscure issues have only shown up on large tests, so yep, I'll add problem size to the parametrization.

Reply:
I hit "Comment" on that one too soon.  None of the tests in PROTO-tests/tests/random_test.py have size as a parameter.  All have sizes hardcoded, usually as 10, but not always.  This would be a good topic for a group discussion.